src/LazyLoad.jsx

```diff
@@ -35,7 +35,11 @@ export default class LazyLoad extends Component {
 
     if (top === 0 || (top <= (scrollTop + window.innerHeight + threshold)
                       && (top + height) > (scrollTop - threshold))) {
-      this.setState({ visible: true });
+      this.setState({ visible: true }, () => {
+        if (onContentVisible) {
```

Comment:
Ah you're absolutely right, I actually just saw that convention used in another module and thought it was better than what I was doing. Thanks for the explanation.